CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## Version 1.1.0 - Feature release - 2023-04-25
+
+- Add support for Python 3.8, 3.9, 3.10, 3.11
```

Comment:
As you didn't make any code/dependency changes to allow support of these versions, I would just say "Add supported python versions ..."  here